account_move_base_import/tests/test_invoice.py

```diff
@@ -0,0 +1,256 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+import datetime as dt
+
+from odoo.modules import get_module_resource
```

Comment:
Still works but seems to be deprecated in favor of get_resource_path : https://github.com/odoo/odoo/blob/04d9715766adab97dd6da66c97e50fd6a30dab31/odoo/modules/module.py#L226